docs/_news/news08.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: news
+title: New for Codewind 0.8.0
+description: New for Codewind 0.8.0
+keywords: news, new, updates, update, version, hybrid
+duration: 1 minute
+permalink: news07
+---
+
+## Codewind 0.8.0
+Thursday 23 January 2020
+
+#### ✨ New Features and Highlights for 0.8.0 ✨
+
+**Appsody**
+- The codewind-appsody-extension now supports passing environment options to the Appsody CLI using [`--docker-options`](https://github.com/eclipse/codewind/issues/1401).
+
+**Che**
+- The [codewind-che-plugin now supports Helm 3](https://github.com/eclipse/codewind/issues/1320) and the insecure non-default tiller was removed.
```

Comment:
-Let's make the `codewind-che-plugin` code phrase format also. I think hyperlinks can work with the code phrase format. If not or if it looks weird, maybe make only the "Helm 3" part the hyperlink?
- Add a comma after the hyperlink to separate the two independent clauses.